js/ui/components/clubhouse.js

```diff
@@ -226,7 +226,11 @@ var ClubhouseNotificationBanner = new Lang.Class({
             Tweener.addTween(this.actor,
                              { x: endX,
                                time: CLUBHOUSE_BANNER_ANIMATION_TIME,
-                               transition: 'easeOutQuad'
+                               transition: 'easeOutQuad',
+                               onComplete: () => {
+                                   // Ensure it only slides in the first time
+                                   this._shouldSlideIn = false;
```

Comment:
I think that to make this even simpler, you could set the variable to `false` right away, without waiting for the tween to complete.